packages/teleterm/src/services/config/providers/keyboardShortcutsConfigProvider.ts

```diff
@@ -50,7 +50,27 @@ export const keyboardShortcutsConfigProvider: ConfigServiceProvider<KeyboardShor
         'toggle-identity': 'Command-I',
       };
 
-      const windowsAndLinuxShortcuts: KeyboardShortcutsConfig = {
+      const windowsShortcuts: KeyboardShortcutsConfig = {
+        'tab-1': 'Ctrl-1',
+        'tab-2': 'Ctrl-2',
+        'tab-3': 'Ctrl-3',
+        'tab-4': 'Ctrl-4',
+        'tab-5': 'Ctrl-5',
+        'tab-6': 'Ctrl-6',
+        'tab-7': 'Ctrl-7',
+        'tab-8': 'Ctrl-8',
+        'tab-9': 'Ctrl-9',
+        'tab-close': 'Ctrl-W',
+        'tab-new': 'Ctrl-T',
+        'tab-previous': 'Ctrl-Shift-Tab',
```

Comment:
@avatus I would be thankful if you could test `tab-previous` and `tab-next` on Windows. For some reason these shortcuts don't work on Parallels (I tested them in Edge).

Reply:
They work great on windows 👍